consensus/broadcast/delayedBroadcast.go

```diff
@@ -637,6 +690,12 @@ func (dbb *delayedBlockBroadcaster) interceptedHeader(_ string, headerHash []byt
 	dbb.cacheHeaders.Put(headerHash, struct{}{}, 0)
 	dbb.mutHeadersCache.Unlock()
 
+	aggSig, bitmap := headerHandler.GetPreviousAggregatedSignatureAndBitmap()
```

Comment:
this will probably not help as it refers to the previous equivalent proof, not the one for the current header

Reply:
discussed, will be fixed in another PR